tools/data-handler/src/command-handler.ts

```diff
@@ -1167,18 +1167,27 @@ export class Commands {
       };
     }
 
-    console.log('Running Cyberismo app on http://localhost:3000/');
-    console.log('Press Control+C to stop.');
-
     // __dirname when running cards ends with /tools/data-handler/dist - use that to navigate to app path
     const baseDir = dirname(fileURLToPath(import.meta.url));
     const appPath = resolve(baseDir, '../../app');
 
     // since current working directory changes, we need to resolve the project path
     const projectPath = resolve(path);
 
-    const args = [`start`, `--project_path="${projectPath}"`];
-    execFileSync(`npm`, args, { shell: true, cwd: `${appPath}` });
+    // Start trying from port 3000 and increase the port number if the port is already in use
+    let port = 3000;
+    let args = ['-s', `start`, `--project_path="${projectPath}"`, '--', `--port ${port}`];
+    
+    while (port < 3099) {
```

Comment:
Rather theoretical issue, but what happens if all 3000 - 3099 are reserved?
To my eyes, it just finishes with "200".